release-notes/opensearch-release-notes-1.0.0-rc1.md

```diff
@@ -0,0 +1,335 @@
+# OpenSearch and Dashboards 1.0.0-rc1 Release Notes
+
+## Release Highlights
+
+## Release Details
+
+OpenSearch and Dashboards 1.0.0-rc1 includes the following OpenSearch Migration, features, enhancements, bug fixes, infrastructure, documentation, maintenance, and refactoring updates.
+
+## BREAKING CHANGES
+
+## FEATURES
+
+### Opensearch Index Management
+* Adds support for Transform feature ([#17](https://github.com/opensearch-project/index-management/pull/17))
+
+
+### Opensearch Index Management Dashboards Plugin
+* Introducing transforms ([#16](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/16))
+* Show data streams in 'Indices' and 'Managed Indices' sections ([#24](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/24))
```

Comment:
I think this PR is not part of this release?

Reply:
Removed